src/snoowrap.js

```diff
@@ -695,270 +728,478 @@ const snoowrap = class snoowrap {
-  * //  PrivateMessage { body: '/u/not_an_aardvark has been invited by /u/actually_an_aardvark to ...', ... }
-  * // ]
-  */
+   * @summary Gets the authenticated user's modmail.
+   * @param {object} [options={}] Options for the resulting Listing
+   * @returns {Promise} A Listing of the user's modmail
+   * @example
+   *
+   * r.getModmail({limit: 2}).then(console.log)
+   * // => Listing [
+   * //  PrivateMessage { body: '/u/not_an_aardvark has accepted an invitation to become moderator ... ', ... },
+   * //  PrivateMessage { body: '/u/not_an_aardvark has been invited by /u/actually_an_aardvark to ...', ... }
+   * // ]
+   */
   getModmail (options = {}) {
     return this._getListing({uri: 'message/moderator', qs: options});
   }
+
+  /**
+   * @summary Gets a list of ModmailConversations from the authenticated user's subreddits.
+   * @param {object} [options] Options for the resulting Listing
+   * @returns {Promise<Listing<ModmailConversation>>} A Listing containing Subreddits
+   * @example
+   *
+   * r.getNewModmailConversations({limit: 2}).then(console.log)
+   * // => Listing [
+   * //  ModmailConversation { messages: [...], objIds: [...], subject: 'test subject', ... },
+   * //  ModmailConversation { messages: [...], objIds: [...], subject: 'test subject', ... }
+   * // ]
+   */
+  getNewModmailConversations (options = {}) {
+    return this._getListing({
+      uri: 'api/mod/conversations', qs: options, _name: 'ModmailConversation', _transform: response => {
+        response.after = null;
+        response.before = null;
+        response.children = [];
+
+        for (const conversation of response.conversationIds) {
+          response.conversations[conversation].participant = this._newObject('ModmailConversationAuthor', {
+            ...response.conversations[conversation].participant
+          });
+          const conversationObjects = objects.ModmailConversation._getConversationObjects(
+            response.conversations[conversation],
+            response
+          );
+          const data = {
+            ...conversationObjects,
+            ...response.conversations[conversation]
+          };
+          response.children.push(this._newObject('ModmailConversation', data));
+        }
+        return this._newObject('Listing', response);
+      }
+    });
+  }
+
+  /**
+   * @summary Create a new moderator discussion
+   * @param {object} options
+   * @param {string} options.body Body of the discussion
+   * @param {string} options.subject Title or subject
+   * @param {string} options.srName Subreddit name without fullname
+   * @returns {Promise<ModmailConversation>} the created ModmailConversation
+   * @example
+   *
+   * r.createModeratorDiscussion({
+   *   body: 'test body',
+   *   subject: 'test subject',
+   *   srName: 'AskReddit'
+   * }).then(console.log)
+   * // ModmailConversation { messages: [...], objIds: [...], subject: 'test subject', ... }
+   */
+  createModeratorDiscussion ({
```

Comment:
Minor: Maybe this should be called `createModmailDiscussion` for consistency with the other functions that use the term `modmail`.

Reply:
Sounds good to me

Reply:
Changed